README.md

```diff
@@ -301,6 +301,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [Punto Switcher](http://punto.yandex.ru/)
   - [PyCharm 4](https://www.jetbrains.com/pycharm/)
   - [PyPI](https://pypi.python.org/pypi)
+  - [PyRadio](https://github.com/coderholic/pyradio)
```

Comment:
I can, but that homepage is a bit behind on documentation / installation instructions. You still want me to?